server-base.R

```diff
@@ -224,7 +224,7 @@ shinyServer(function(input, output, session){
   observe({
     region$replot
     input$map_base
-    show_zone_popup <- isolate({input$line_type == 'none'})
```

Comment:
@nikolai-b 
Have double-checked it. At my end, it's not always `false`. This is true when `none` is selected (Cycling Flows) - and zones are clickable. Otherwise, zones are not clickable.